### PR TITLE
clean out misc

### PR DESCRIPTION
**Pull request by sannecottaar**
Cleaned out files in misc/ref that weren't used.
Removed pyrolite_uncertainty.py in misc folder
moved colors.py and helpers_solid_solution.py to contrib/CHRU2014 (only place it is used)
added readme to contrib/CHRU2014

### CODE REVIEW

**Comment by bobmyhill** [comment]
@sannecottaar, I deal with the deprecated model.py stuff in #427, so you don't need to worry about removing that (the cause of your failing tests).

File: contrib/CHRU2014/paper_averaging.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 import burnman
 from burnman import minerals
-import misc.colors as colors
+import .colors as colors
```
Comment (bobmyhill) [inline contrib/CHRU2014/paper_averaging.py:45]: I think `from contrib.CHRU2014 import colors` would work (here and in other scripts).

**Comment by bobmyhill** [comment]
Ooops, `colors.py` is deleted from `misc/`, but not yet added into `contrib/CHRU2014/`. After that, everything *should* work :)

File: misc/ref/example_solution_creation_and_manipulation.py.out [hunk 2]
```diff
@@ -1,156 +0,0 @@
-Endmember occupancies for bridgmanite in the FMASO system
```
Comment (bobmyhill) [inline misc/ref/example_solution_creation_and_manipulation.py.out:1]: I think this file needs to be restored.

Comment (sannecottaar) [reply misc/ref/example_solution_creation_and_manipulation.py.out:1]: I don't see the example?

Comment (bobmyhill) [reply misc/ref/example_solution_creation_and_manipulation.py.out:1]: It's in contrib/solution_polytope. It's not a perfect name, but it's the name given in the published paper.

**Pull request merged**